.github/workflows/masterfhirvalidation.yml

```diff
@@ -92,7 +73,7 @@ jobs:
       - name: Run FHIR Validator
         run: 
           nohup java -jar validation-service-fhir-r4/target/fhir-validator.jar --terminology.url=https://ontology.nhs.uk/production1/fhir --terminology.authorization.tokenUrl=https://ontology.nhs.uk/authorisation/auth/realms/nhs-digital-terminology/protocol/openid-connect/token --terminology.authorization.clientId=${{ secrets.ONTO_CLIENT_ID }} --terminology.authorization.clientSecret=${{ secrets.ONTO_CLIENT_SECRET }} --aws.validationSupport=false --aws.queueEnabled=false &
-          sleep 120
+          sleep 120 #used to allow time for the server to run before starting test
         
       - name: Run Test
         run: make -C validation test
```

Comment:
Suggestion above would make this `make test` as opposed to `make -C validation test`, same with others